frontend/src/Components/Toolbar/Filters/FilterRangeSlider.tsx

```diff
@@ -21,7 +21,7 @@ export function FilterRangeSlider({ varName }: { varName: NumberArrayKeys<Filter
   }, [store.filtersStore.filterValues[varName]]);
 
   return useObserver(() => {
-    const initial = store.filtersStore.initialFilterValues[varName];
+    const initial = (store.filtersStore.initialFilterValues as any)[varName];
```

Comment:
This is a real downside. We're losing all the power of typescript when we force an `any`, opening us up to more bugs.

Please take a closer look at these `any` changes and see how you can reduce the number of them